signalfx/resource_signalfx_detector.go

```diff
@@ -20,6 +20,62 @@ const (
 	DetectorAppPath = "/detector/"
 )
 
+var (
+	detectorRuleSchema = map[string]*schema.Schema{
+		"severity": {
+			Type:         schema.TypeString,
+			Required:     true,
+			ValidateFunc: validateSeverity,
+			Description:  "The severity of the rule, must be one of: Critical, Warning, Major, Minor, Info",
+		},
+		"detect_label": {
+			Type:        schema.TypeString,
+			Required:    true,
+			Description: "A detect label which matches a detect label within the program text",
+		},
+		"description": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Description of the rule",
+		},
+		"notifications": {
+			Type:     schema.TypeList,
+			Optional: true,
+			Elem: &schema.Schema{
+				Type:         schema.TypeString,
+				ValidateFunc: validateNotification,
+			},
+			Description: "List of strings specifying where notifications will be sent when an incident occurs. See https://developers.signalfx.com/v2/docs/detector-model#notifications-models for more info",
+		},
+		"disabled": {
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Default:     false,
+			Description: "(default: false) When true, notifications and events will not be generated for the detect label",
+		},
+		"parameterized_body": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Custom notification message body when an alert is triggered. See https://developers.signalfx.com/v2/reference#detector-model for more info",
+		},
+		"parameterized_subject": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Custom notification message subject when an alert is triggered. See https://d    evelopers.signalfx.com/v2/reference#detector-model for more info",
```

Comment:
looks like there are some spaces inside the url

Reply:
Good catch. Fixed